### PR TITLE
Tutorial review

### PR DESCRIPTION
**Pull request by yhontyk**
My proposal for some light edits in the first tutorial. 

One note on headings - we recommend using sentence capitalization for all headings https://docs.ubuntu.com/styleguide/en

I was also thinking of changing "Learn What Mir Can Do" to "Getting started with Mir". I'm not sure if it's the best choice but it sounds like a basic enough first tutorial that any user should get themselves acquainted with.

I think the tutorial is accessible enough as is so I didn't introduce any significant changes to steps themselves.

Jira: https://warthogs.atlassian.net/browse/DOCPR-852

### CODE REVIEW

File: doc/sphinx/tutorial/index.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Tutorials
 These pages provide first-time introductions to key Mir aspects
 
-- [Learn What Mir Can Do](/tutorial/learn-what-mir-can-do): A showcase of Mir's capabilities
+- [Getting started with Mir](/tutorial/learn-what-mir-can-do): A showcase of Mir's capabilities
```
Comment (AlanGriffiths) [inline doc/sphinx/tutorial/index.md:4]: We should probably rename the slug to match

File: doc/sphinx/tutorial/learn-what-mir-can-do.md [hunk 2]
```diff
@@ -1,135 +1,129 @@
...
-The main script you'll want to play around with is `miral-app`. It runs a shell
-with some eyecandy by default, but you can run it in kiosk mode if that fits
-your usecase more.
+You will work with a `miral-app` script that runs a shell with basic GUI.
 
-For the uninitiated: The shell mode runs a floating window manager just like
-GNOME and such where you can move windows and maximize or minimize them. Kiosk
-mode on the other hand assumes you want one (or more) applications in
-fullscreen mode all the time.
+Mir allows you to run applications in the *shell mode* or in *kiosk mode*.
```
Comment (AlanGriffiths) [inline doc/sphinx/tutorial/learn-what-mir-can-do.md:34]: It comes from the pre-existing text but "shell mode" is a bad term: all the modes are types of shell.

```suggestion
Mir allows you to run a graphical shell in *desktop mode* or in *kiosk mode*.
```

File: doc/sphinx/tutorial/learn-what-mir-can-do.md [hunk 3]
```diff
@@ -1,135 +1,129 @@
...
 
-To run in shell mode, just run:
+Shell mode means that the application is opened in a floating window manager and you can move the window around the screen, maximize or minimize it.
+Kiosk mode means that the application is opened in fullscrean mode.
```
Comment (AlanGriffiths) [inline doc/sphinx/tutorial/learn-what-mir-can-do.md:37]:
```suggestion
Kiosk mode means that the application is opened in fullscreen mode. 
```

File: doc/sphinx/tutorial/learn-what-mir-can-do.md [hunk 4]
```diff
@@ -1,135 +1,129 @@
...
+You will work with a `miral-app` script that runs a shell with basic GUI.
 
-For the uninitiated: The shell mode runs a floating window manager just like
-GNOME and such where you can move windows and maximize or minimize them. Kiosk
-mode on the other hand assumes you want one (or more) applications in
-fullscreen mode all the time.
+Mir allows you to run applications in the *shell mode* or in *kiosk mode*.
 
-To run in shell mode, just run:
+Shell mode means that the application is opened in a floating window manager and you can move the window around the screen, maximize or minimize it.
```
Comment (AlanGriffiths) [inline doc/sphinx/tutorial/learn-what-mir-can-do.md:36]:
```suggestion
Desktop mode means that application windows are opened are "floating and you can move them around the screen, maximize or minimize them.
```

File: doc/sphinx/tutorial/learn-what-mir-can-do.md [hunk 5]
```diff
@@ -1,135 +1,129 @@
...
+To run the script in the kiosk mode:
 ```sh
 miral-app -kiosk
 ```
 
-**Note**: By default, Mir does not enable Wayland extensions that normal
-applications should not be using. For demonstration purposes we will override
-this and allow all supported extensions in some of the following examples by
-passing `--add-wayland-extensions all` when running the example.
+## Running applications natively
```
Comment (AlanGriffiths) [inline doc/sphinx/tutorial/learn-what-mir-can-do.md:49]:
```suggestion
## Running shells natively
```

File: doc/sphinx/tutorial/learn-what-mir-can-do.md [hunk 6]
```diff
@@ -1,135 +1,129 @@
...
 
-## Running Natively
-The previous section showed how you can run Mir demos under an X11 or Wayland
-session. But Mir compositors can also run "natively" by launching them from a
-virtual terminal or a greeter.
+Mir compositors supports running application natively by launching them from a virtual terminal or a login screen.
```
Comment (AlanGriffiths) [inline doc/sphinx/tutorial/learn-what-mir-can-do.md:51]:
```suggestion
Mir compositors supports running shells natively by launching them from a virtual terminal or a login screen.
```

File: doc/sphinx/tutorial/learn-what-mir-can-do.md [hunk 7]
```diff
@@ -1,135 +1,129 @@
...
 
 ### Launching from a virtual terminal
-To switch to a virtual terminal, you can press CTRL+ALT+F\<Number\>. You can then
-log in and run:
+
+1. Switch to a virtual terminal by pressing CTRL+ALT+F\<Number\> and log into the virstual environemtn.
```
Comment (AlanGriffiths) [inline doc/sphinx/tutorial/learn-what-mir-can-do.md:55]:
```suggestion
1. Switch to a virtual terminal by pressing CTRL+ALT+F\<Number\> and log in.
```

File: doc/sphinx/tutorial/learn-what-mir-can-do.md [hunk 8]
```diff
@@ -1,135 +1,129 @@
...
 
-## Using On-screen Keyboards
-Mir based compositors can easily support on-screen keyboards. Note that due to
-security reasons, the Wayland extensions needed are disabled by default.
+## Using on-screen keyboards
+Mir-based compositors support on-screen keyboards. 
 
-For the purpose of demonstation, we'll use `ubuntu-frame-osk`, but you're free
-to use any Wayland compatible on-screen keyboard.
+**Note**: Due to security reasons, the Wayland extensions are disabled by default. In this tutorial, you will override this setting by
```
Comment (AlanGriffiths) [inline doc/sphinx/tutorial/learn-what-mir-can-do.md:69]:
```suggestion
**Note**: Due to security reasons, some Wayland extensions needed by on-screen keyboards are disabled by default. In this tutorial, you will override this by
```

**Review by AlanGriffiths — CHANGES_REQUESTED** [review]
Thanks. There are some subject-matter corrections but this is a helpful push

**Review by tarek-y-ismail — CHANGES_REQUESTED** [review]

File: doc/sphinx/tutorial/getting-started-with-mir.md [hunk 9]
```diff
@@ -0,0 +1,129 @@
+# Getting started with Mir
+
+This tutorial will guide you through Mir's basic functionality. 
+
+By the end of the tutorial, you will install and run a demo application, learn how to use Mir in different environments, and learn about features that Mir provides for Mir-based compositors. 
+
+If you are unfamiliar with Mir, start with this tutorial, and then proceed to a [developer tutorial](write-your-first-wayland-compositor) which will guide you through the process of writing compositors.
```
Comment (tarek-y-ismail) [inline doc/sphinx/tutorial/getting-started-with-mir.md:7]:
```suggestion
If you are unfamiliar with Mir, start with this tutorial, and then proceed to a [developer tutorial](write-your-first-wayland-compositor) which will guide you through the process of writing a Mir compositor.
```

File: doc/sphinx/tutorial/getting-started-with-mir.md [hunk 10]
```diff
@@ -0,0 +1,129 @@
...
+### On Alpine
+```sh
+sudo apk add mir-demos mir
+```
+
+## Running applications on X11 or Wayland
+
+We'll work with `miral-app`, a script that handles running a Mir shell with a basic GUI.
+
+Mir allows you to run a graphical shell in *desktop mode* or in *kiosk mode*.
```
Comment (tarek-y-ismail) [inline doc/sphinx/tutorial/getting-started-with-mir.md:34]:
```suggestion
`miral-app` allows you to run a graphical shell in *desktop mode* or in *kiosk mode*, among other modes. Mir also gives you the ability to write your own mode.
```

File: doc/sphinx/tutorial/getting-started-with-mir.md [hunk 11]
```diff
@@ -0,0 +1,129 @@
...
+
+Desktop mode means that application windows that are opened are floating, you can move them around the screen, maximize, or minimize them.
+Kiosk mode means that the application is opened in fullscreen mode. You are unable to move or resize it.
+
+To run the script in shell mode:
```
Comment (tarek-y-ismail) [inline doc/sphinx/tutorial/getting-started-with-mir.md:39]:
```suggestion
To run the script in desktop mode:
```

File: doc/sphinx/tutorial/getting-started-with-mir.md [hunk 12]
```diff
@@ -0,0 +1,129 @@
...
+1. Switch to a virtual terminal by pressing CTRL+ALT+F\<Number\> and log in.
+
+2. Run the script:
+```sh
+miral-app
+```
+
+### Launching from a login screen
+
+Open the window manager list, start **Mir Shell**, log in and `miral-shell` will be running fullscreen.
```
Comment (tarek-y-ismail) [inline doc/sphinx/tutorial/getting-started-with-mir.md:64]:
```suggestion
Open the window manager list, choose **Mir Shell**, log in, and `miral-shell` will be running fullscreen.
```

File: doc/sphinx/tutorial/getting-started-with-mir.md [hunk 13]
```diff
@@ -0,0 +1,129 @@
...
+
+## Using on-screen keyboards
+Mir-based compositors support on-screen keyboards. 
+
+**Note**: Due to security reasons, some Wayland extensions needed by on-screen keyboards are disabled by default. In this tutorial, we override this setting by
```
Comment (tarek-y-ismail) [inline doc/sphinx/tutorial/getting-started-with-mir.md:69]:
```suggestion
**Note**: Due to security reasons, some Wayland extensions needed by on-screen keyboards are disabled by default. In this tutorial, we override this setting by passing `--add-wayland-extensions all` when launching `miral-app`.
```

File: doc/sphinx/tutorial/getting-started-with-mir.md [hunk 14]
```diff
@@ -0,0 +1,129 @@
...
+## Remote desktop
+Mir supports remote desktops via the VNC protocol. To demo this, you'll use `wayvnc` - a VNC server. 
+
+1. Install `wayvnc`:
+
+```sh
+sudo apt install wayvnc
+```
+
+2. Install your preferred [VNC client](https://help.ubuntu.com/community/VNC/Clients).
```
Comment (tarek-y-ismail) [inline doc/sphinx/tutorial/getting-started-with-mir.md:116]: The VNC client list has been last updated over two years ago, should we contact someone to review it?

File: doc/sphinx/tutorial/getting-started-with-mir.md [hunk 15]
```diff
@@ -0,0 +1,129 @@
...
+miral-app --add-wayland-extensions all
+```
+
+4. In the shell, open the terminal and run `wayvnc`:
+```sh
+wayvnc
+```
+A `wayvnc` server will start and will listen to `localhost`. 
+
+5. Run your VNC client and connect to `localhost`. You will see the exact view in both Mir compositor and the VNC viewer.
```
Comment (tarek-y-ismail) [inline doc/sphinx/tutorial/getting-started-with-mir.md:129]:
```suggestion
5. Run your VNC client and connect to `localhost`. You will see the exact same view in both the Mir compositor and the VNC viewer. 
```

**Review by tarek-y-ismail — CHANGES_REQUESTED** [review]

File: doc/sphinx/tutorial/getting-started-with-mir.md (hunk 9 above)
Comment (AlanGriffiths) [reply doc/sphinx/tutorial/getting-started-with-mir.md:7]: I don't think we need to repeat "Mir"...

```suggestion
If you are unfamiliar with Mir, start with this tutorial, and then proceed to a [developer tutorial](write-your-first-wayland-compositor) which will guide you through the process of writing a compositor.
```

File: doc/sphinx/tutorial/getting-started-with-mir.md (hunk 10 above)
Comment (AlanGriffiths) [reply doc/sphinx/tutorial/getting-started-with-mir.md:34]: stet. (This tutorial is about running the examples, not writing bespoke compositors.)

**Review by AlanGriffiths — CHANGES_REQUESTED** [review]

**Review by tarek-y-ismail — APPROVED** [review]

File: doc/sphinx/tutorial/getting-started-with-mir.md [hunk 16]
```diff
@@ -0,0 +1,129 @@
...
+sudo snap install ubuntu-frame-osk
+sudo snap connect ubuntu-frame-osk:wayland
+```
+
+2. Start `miral-app`:
+```sh
+miral-app --add-wayland-extensions all
+```
+
+3. Once the shell loads, start the terminal and start your on-screen keyboard:
```
Comment (AlanGriffiths) [inline doc/sphinx/tutorial/getting-started-with-mir.md:85]:
```suggestion
3. Once the shell loads, start the terminal (with `Ctrl-Alt-Shift-T`) and start your on-screen keyboard:
```

File: doc/sphinx/tutorial/getting-started-with-mir.md (hunk 14 above)
Comment (AlanGriffiths) [reply doc/sphinx/tutorial/getting-started-with-mir.md:116]: If you think there's something needs changing...  it's a Wiki

**Review by AlanGriffiths — CHANGES_REQUESTED** [review]
More nitpicking for clarity

**Review by AlanGriffiths — APPROVED** [review]

**Pull request merged**